draft-ietf-quic-reliable-stream-reset.md

```diff
@@ -87,29 +87,30 @@ remember the value of this transport parameter.  If 0-RTT data is accepted by
 the server, the server MUST not disable this extension on the resumed
 connection.
 
-# RELIABLE_RESET_STREAM Frame
+# CLOSE_STREAM Frame
 
-Conceptually, the RELIABLE_RESET_STREAM frame is a RESET_STREAM frame with an
+Conceptually, the CLOSE_STREAM frame is a RESET_STREAM frame with an
 added Reliable Size field.
 
 ~~~
-RELIABLE_RESET_STREAM Frame {
-  Type (i) = 0x72,
+CLOSE_STREAM Frame {
+  Type (i) = 0x20..0x21,
   Stream ID (i),
-  Application Protocol Error Code (i),
+  [Application Protocol Error Code (i)],
```

Comment:
I would prefer if this were unconditional.  A value of 0 (or whatever application error equates to no error) is fine.

I see why you made this choice, but if the goal is to send STREAM+FIN, then STREAM+FIN exists.  This frame only makes sense if reliable size < final size.